src/util/sss_log.c

```diff
@@ -26,10 +26,15 @@
 
 #ifdef WITH_JOURNALD
 #include <systemd/sd-journal.h>
-#else  /* WITH_JOURNALD */
+#else
+#ifdef WITH_STDERR_SYSLOG
```

Comment:
This can be simplified with an `if defined` structure:
```
#if defined(WITH_JOURNALD)
#include <systemd/sd-journal.h>
#elif defined(WITH_STDERR_SYSLOG)
#include <stdio.h>
#else
#include <syslog.h>
#endif
```